### ISSUE TITLE
v5.2.0-alpha.0 feedback

### ISSUE THREAD

**Issue by andyrichardson** [react-cosmos/react-cosmos#1176]
## About

Tried out the alpha and statically, everything looks to work great!

Super minor but would be awesome if the export the FixtureApi type for times where explicit typing is required (i.e. a reduce). I used this as a workaround - `type FixtureApi = ReturnType<typeof getFixtures2>[number];`

I'm now [getting a runtime error in jest](https://app.circleci.com/pipelines/github/FormidableLabs/urql-devtools/551/workflows/01917f4e-0404-43ea-a4d6-23d6fb4dbe4c/jobs/2854) after updating and using _getFIxtures2_. Looks like [fixtureTree.js](https://www.runpkg.com/?react-cosmos-shared2@5.2.0-alpha.0/fixtureTree.js) is not being transpiled to commonjs [like other files](https://www.runpkg.com/?react-cosmos-shared2@5.2.0-alpha.0/fixtureState.js).

Hope this feedback is useful! You can [find the diff for our update here](https://github.com/FormidableLabs/urql-devtools/pull/147/files#diff-805f2a2176010275f3a22eb673f51c8f).

**Comment by skidding** [comment]
Great feedback!

Fixed the `fixtureTree` entry point and exported the `FixtureApi` type in `5.2.0-alpha.2`.

**Comment by andyrichardson** [comment]
@skidding thanks for this! 

One final thing before I close this, I think the _rendererUrl_ and _playgroundUrl_ values are the wrong way round. Right now I'm seeing the following:

```js
{
  rendererUrl: 'http://cosmos:5000/?fixtureId=%7B%22path%2...',
  playgroundUrl: 'http://cosmos:5000/_renderer.html?_fixtureId...'
}
```

**Comment by skidding** [comment]
![image](https://user-images.githubusercontent.com/250750/79332565-b9721380-7f25-11ea-8624-a95fc88fa661.png)

😂

Fixed in `5.2.0-alpha.5`.

@andyrichardson thanks again!
